app/components/more-details.jsx

```diff
@@ -0,0 +1,139 @@
+  const {
+    point = { subject: '', description: '', _id: '' },
+    defaultValue = { subject: '', description: '' },
+    onDone = () => {},
+    className,
+    ...otherProps
+  } = props
+  const [data, setData] = useState(initialData)
+  const classes = useStyles()
+  const initialData = {}
+
+  const schema = {
+    type: 'object',
+    properties: {
+      householdIncome: {
+        title: 'Household Income',
+        type: 'string',
+        enum: ['0-10000', '10000-20000', '20000-30000'],
+        default: 'Select household range',
+      },
+      housing: {
+        title: 'Housing',
+        type: 'string',
+        enum: ['House', 'Apartment', 'None'],
+      },
+      numberOfSiblings: {
+        title: 'Number of Siblings',
+        type: 'string',
+        enum: ['0', '1', '2'],
+      },
+      submit: {
+        type: 'button',
+        title: 'Submit',
+      },
+    },
+  }
+  const uischema = {
+    type: 'VerticalLayout',
+    elements: [
+      {
+        type: 'Control',
+        scope: '#/properties/householdIncome',
+        options: {
+          classNames: classes.stackedControl,
+        },
+      },
+
+      {
+        type: 'Control',
+        scope: '#/properties/housing',
+        labelAlignment: 'top',
+      },
+      {
+        type: 'Control',
+        scope: '#/properties/numberOfSiblings',
+        labelAlignment: 'top',
+      },
+      {
+        type: 'Button',
+        scope: '#/properties/submit',
+      },
+    ],
+  }
+
+  const handleOnDone = ({ valid, value }) => {
+    console.log('handleOnDone clicked')
+    // value.parentId = `${point._id}`
+    // onDone({ valid, value })
+  }
+
+  const onSubmit = data => {
+    console.log('Form data:', data)
+    console.log('onSubmit clicked')
+    // Handle form submission logic here
+  }
+
+  return (
+    <div
+      className={cx(classes.container, className)}
+      {...otherProps}
+    >
+      <h3>We just need a few more details about you to get started.</h3>
+      <div className={classes.formContainer}>
+        <Column>
+          <JsonForms
+            schema={schema}
+            uischema={uischema}
+            data={data}
+            renderers={vanillaRenderers}
+            details={} // the data for jsonforms
+            cells={vanillaCells}
+            onChange={({ data, _errors }) => setData(data)}
+            onSubmit={onSubmit}
+          />
+          <Submit className={classes.submitButton}>Submit</Submit>
+        </Column>
+      </div>
+    </div>
+  )
+}
+
+const useStyles = createUseStyles(theme => ({
+  container: {
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    fontFamily: 'Inter',
+    // border: 'medium dashed green',
+  },
+  formContainer: {
+    margin: '1.25rem',
+    // border: 'medium solid red'
+  },
+  selectList: {},
+  selectTitle: { fontWeight: 'bold' },
+  submitButton: { backgroundColor: '#06335C', color: '#fff', borderRadius: '10px', width: '100%' },
```

Comment:
Never use px.  Use 10/16 to get the number of rem.  This makes the component more responsive, because when you get onto mobile devices, the default font size is more like 8 or 12. 

Please check for other uses of px too.